src/feature/stats/rephist.c

```diff
@@ -1122,15 +1082,22 @@ static bw_array_t *dir_read_array = NULL;
     directory protocol. */
 static bw_array_t *dir_write_array = NULL;
 
-/** Set up [dir_]read_array and [dir_]write_array, freeing them if they
- * already exist. */
-static void
-bw_arrays_init(void)
+/* Free up [dir_]read_array and [dir_]write_array. */
+STATIC void
+bw_arrays_destroy(void)
```

Comment:
We call our free functions "*_free" or "*_free_all".

Reply:
I will correct every typo in my last commit.